app/src/main/java/ru/chaichuk/hwapp/fragments/MoviesDetailsFragment.kt

```diff
@@ -0,0 +1,32 @@
+package ru.chaichuk.hwapp.fragments
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import ru.chaichuk.hwapp.R
+
+class MoviesDetailsFragment : Fragment() {
```

Comment:
instead of override `onCreateView` you can write like so
class MoviesDetailsFragment : Fragment(R.layout.fragment_movies_details)

Reply:
did so

Reply:
good